config/development.json

```diff
@@ -86,6 +86,7 @@
 		"manage/pages": true,
 		"manage/payment-methods": true,
 		"manage/people": true,
+		"manage/people/invites": true,
 		"manage/people/readers": true,
 		"manage/people/role-filtering": true,
```

Comment:
It looks like we could (separately) clean up some of these feature flags - from what I can see, `manage/people` and `manage/people/readers` are always enabled.  `manage/people/role-filtering` seems like it may be stalled (https://github.com/Automattic/wp-calypso/pull/9264).